test/errors/useImportUnknownSym-detailed.good

```diff
@@ -1,5 +1,5 @@
 ─── error in useImportUnknownSym.chpl:6 [UseImportUnknownSym] ───
-  Cannot find symbol 'b' for import.
+  Cannot 'import' symbol 'b' as it is not defined in 'M'.
```

Comment:
I wouldn't feel the need to quote 'import' here since it works fine as an English word (and I think we're specifically trying to say the English word here, not refer the reader to the keyword).

Reply:
I think here, I went with symmetry; in particular, for _use statements_, this would read `Cannot use symbol 'b'`, and there, the common meaning of `use` differs from the Chapel meaning of `use`. Thus, it made sense to quote the word `use` to make it clear that the Chapel meaning is used. Thus, I quote `import` too, to make the error messages symmetric.

Reply:
I agree 'use' should be quoted for clarity and lack of ambiguity.  My preference would still be to not quote 'import' because it isn't nearly as ambiguous and scans to me more like air-quotes ("a laser").